clang/include/clang/AST/DeclID.h

```diff
@@ -124,6 +130,15 @@ class DeclIDBase {
 
   bool isInvalid() const { return ID == PREDEF_DECL_NULL_ID; }
 
+  unsigned getModuleFileIndex() const { return ID >> 32; }
+
+  unsigned getLocalDeclIndex() const {
+    // Implement it directly instead of calling `llvm::maskTrailingOnes` since
+    // we don't want `MathExtras.h` to be inclued here.
```

Comment:
Why not? Just to keep the amount of code lower? Can't we move the function body to an implementation file?

Reply:
Maybe I just didn't want to create an implementation file for such a single simple function. But I just realized I can append it in `DeclBase.cpp` just like many other classes did. So done.